### PR TITLE
feat(WebexMeeting): set meetingID as component prop

### PR DESCRIPTION
**Pull request by alinasuciu**
Update meeting component to use meetingID instead of meetingDestination
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-223897

### CODE REVIEW

File: src/components/WebexMeeting/WebexMeeting.jsx [hunk 1]
```diff
@@ -72,8 +72,7 @@ WebexMeeting.defaultProps = {
    * @param {boolean} isActive  Whether or not the meeting is active
    */
   // eslint-disable-next-line no-confusing-arrow
-  controls: (isActive) =>
-    isActive
-      ? ['mute-audio', 'mute-video', 'share-screen', 'leave-meeting']
-      : ['mute-audio', 'mute-video', 'join-meeting'],
+  controls: (isActive) => isActive
```
Comment (cipak) [inline src/components/WebexMeeting/WebexMeeting.jsx:75]: let's not change this if the only change is code formatting

File: src/components/WebexMeeting/WebexMeeting.jsx [hunk 2]
```diff
@@ -8,17 +8,17 @@ import WebexInterstitialMeeting from '../WebexInterstitialMeeting/WebexInterstit
...
 import WebexMeetingControls from '../WebexMeetingControl/WebexMeetingControls';
 import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
-import {useMeetingDestination} from '../hooks';
+import {useMeeting} from '../hooks';
 
 /**
  * Webex Meeting component displays the default Webex meeting experience.
  *
- * @param {string} props.meetingDestination  ID of the virtual meeting location
+ * @param {string} props.meetingID  ID of the meeting
```
Comment (cipak) [inline src/components/WebexMeeting/WebexMeeting.jsx:16]: the double space was there before, I know, but a single space should be sufficient. Or maybe a dash to make it more readable https://jsdoc.app/tags-param.html

for now I would go with a single space like in the rest of the project.

Comment (alinasuciu) [reply src/components/WebexMeeting/WebexMeeting.jsx:16]: all the files of this kind have double space between the parameter name and description

Comment (cipak) [reply src/components/WebexMeeting/WebexMeeting.jsx:16]: ok missed that

**Review by lalli-flores — CHANGES_REQUESTED** [review]
Can you update the [WebexMeeting README](https://github.com/webex/components/blob/e03f64bd572bc10a4b46e8b4379064bf9b5aae60/src/components/WebexMeeting/README.md#embed) with the prop change?

Also, this would technically be a breaking change because a meeting ID is a different  type of value from receiving a meeting destination. While from the JSON data perspective these look the same, with the SDK adapter they are totally different. Anyone out there using `WebexMeeting` should be made aware of the change via a (generated) version bump.

With that said, I don't want to bump the major version because this is technically a "beta" product, so a minor bump using `feat` in the commit instead of `refactor` should do

**Review by lalli-flores — APPROVED** [review]
Looks good! 👍🏼

**Pull request merged**